### PR TITLE
3.0.0.Alpha5 blog post

### PR DESCRIPTION
**Pull request by maxandersen**
in progress blog for alpha5

### CODE REVIEW

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+tags: release
+synopsis: 'This version is the second biggest update with major impact for Quarkus 3 - upgrading Hibernate 5 to Hibernate 6.'
+author: maxandersen
+---
+
+We are in the process of getting Quarkus 3 ready - last we got our main branch to Jakarta EE 10 to let us getting more things integrated and tested faster.
+
+The big one this time is the move from Hibernate 5 to Hibernate 6 - which *will* cause breakages thus please read the section below for details if you are Hibernate ORM or Hibernate Reactive user.
+
+  * https://github.com/quarkusio/quarkus/pull/31235[Hibernate ORM 6] & Hibernate Reactive 2
```
Comment (yrodiere) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:14]: Not sure the PRs would be very useful to application developers, especially such large PRs...

In any case, if you want those links, you could add the HR one:

```suggestion
  * https://github.com/quarkusio/quarkus/pull/31235[Hibernate ORM 6] & https://github.com/quarkusio/quarkus/pull/31454[Hibernate Reactive 2]
```

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  * `@Scheduled` https://github.com/quarkusio/quarkus/pull/31343[support for timezones]
+  * Support https://github.com/quarkusio/quarkus/pull/31192[HTTP/2 connections] in REST Client Reactive
+  * `@ClientRedirectHandler` https://github.com/quarkusio/quarkus/pull/31142[to support custom redirect in REST Client Reactive]
+  * Add support for https://github.com/quarkusio/quarkus/pull/31159[Gradle 8]
+  * https://github.com/quarkusio/quarkus/pull/31079[Multiple OIDC code flows now allowed in same browser]
+    
+  
+== Hibernate ORM 6
+
+Hibernate ORM 6 is a major upgrade to the main persistence layer of Quarkus. It brings a lot of improvements and new features, but also some breaking changes. We have done our best to make the migration as smooth as possible, but some changes are unavoidable.
```
Comment (yrodiere) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:28]: We're going to do more! Hopefully.

```suggestion
Hibernate ORM 6 is a major upgrade to the main persistence layer of Quarkus. It brings a lot of improvements and new features, but also some breaking changes. We will do our best to make the migration as smooth as possible, but some changes are unavoidable. 
```

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+  * `@ClientRedirectHandler` https://github.com/quarkusio/quarkus/pull/31142[to support custom redirect in REST Client Reactive]
+  * Add support for https://github.com/quarkusio/quarkus/pull/31159[Gradle 8]
+  * https://github.com/quarkusio/quarkus/pull/31079[Multiple OIDC code flows now allowed in same browser]
+    
+  
+== Hibernate ORM 6
+
+Hibernate ORM 6 is a major upgrade to the main persistence layer of Quarkus. It brings a lot of improvements and new features, but also some breaking changes. We have done our best to make the migration as smooth as possible, but some changes are unavoidable. 
+
+We've written https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration[Hibernate ORM 5 to 6 migration] guide for Quarkus users to help you with the migration.
```
Comment (yrodiere) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:30]: We're going to do more! Hopefully.

```suggestion
We've started a https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration[Hibernate ORM 5 to 6 migration] guide for Quarkus users to help you with the migration.
```

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+Take particular note of the https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration#database-orm-compatibility["Best-effort Hibernate ORM 5.6 compatability switch"] which reduces the impact of the default schema changes in Hibernate 6.
```
Comment (yrodiere) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:32]: Maybe let's be clear that this switch is not intended for long-term use?

```suggestion
Take particular note of the https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.0:-Hibernate-ORM-5-to-6-migration#database-orm-compatibility["Best-effort Hibernate ORM 5.6 compatability switch"] which can temporarilly reduce the impact of the default schema changes in Hibernate 6 and give you more time to migrate.
```

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+TBD: MTA command
+
+We want to improve and simplify these migration steps in preparation for Quarkus 3 thus do please report migration issues you encounter so we can improve the migration documentation and scripts.
```
Comment (yrodiere) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:36]:
```suggestion
We want to improve and simplify these migration steps in preparation for Quarkus 3 thus do please https://github.com/quarkusio/quarkus/issues/new/choose[report] migration issues you encounter so we can improve the migration documentation and scripts.
```

**Review by yrodiere — COMMENTED** [review]
A few minor comments about wording.

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc (hunk 4 above)
Comment (maxandersen) [reply _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:32]: slightly reduced it t not make it a too complex sentence to parse :)

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+
+== New Dev UI
+
+Quarkus 2 introduced the Dev UI which provides a web ui to use during development. Extensions could provide their own Dev UI pages to provide additional functionality.
+
+Quarkus 3 now has a new Dev UI which is more extensible and easier to use. It also has a new look and feel.
+
+For now the new dev-ui is served at http://localhost:8080/q/dev-ui
+
+The old dev ui is still the default as not all extensions have migrated.
```
Comment (gsmet) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:71]:
```suggestion
The old Dev UI is still the default as not all extensions have migrated.
```

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+
+We have a https://www.youtube.com/watch?v=sz5ihmA4gaE&list=PLsM3ZE5tGAVbyncLm7ue2V25cwFck7ew9[playlist] on Quarkus YouTube channel that shows off the new Dev UI. Explains the new features and how to use and extend it.
```
Comment (gsmet) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:73]:
```suggestion
We have a https://www.youtube.com/watch?v=sz5ihmA4gaE&list=PLsM3ZE5tGAVbyncLm7ue2V25cwFck7ew9[playlist] on Quarkus YouTube channel that shows off the new Dev UI, explains the new features and how to use and extend it.
```

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+Quarkus CLI and the Maven and Gradle plugins now have the ability to `deploy`.
+
+[source,bash]
+----
+quarkus deploy
+mvn quarkus:deploy
+gradle deploy
+----
+
+This enables the deployment of Quarkus application to platforms like Kubernetes, knative and Openshfit. All without requiring changes to the project dependencies or configuration and therefore simplfying developer experience.
```
Comment (gsmet) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:86]:
```suggestion
This enables the deployment of Quarkus applications to platforms like Kubernetes, Knative and OpenShift. All without requiring changes to the project dependencies or configuration and therefore simplifying developer experience.
```

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 9]
```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: 'Quarkus 3.0.0.Alpha5 released - now with Hibernate 6, new Developer Console and more!'
```
Comment (gsmet) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:3]:
```suggestion
title: 'Quarkus 3.0.0.Alpha5 released - now with Hibernate ORM 6, new Dev UI and more!'
```

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 10]
```diff
@@ -0,0 +1,141 @@
...
+date: 2023-02-18
+tags: release
+synopsis: 'This version is the second biggest update with major impact for Quarkus 3 - upgrading Hibernate 5 to Hibernate 6.'
```
Comment (gsmet) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:6]:
```suggestion
synopsis: 'This version is the second biggest update with major impact for Quarkus 3 - upgrading Hibernate ORM 5 to Hibernate ORM 6.'
```

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 11]
```diff
@@ -0,0 +1,141 @@
...
+And there is a new and improved Dev UI!
+
+image::newdevui.png[]
+
+The list of highlights for this release is:
+
+  * https://github.com/quarkusio/quarkus/pull/31235[Hibernate ORM 6] & https://github.com/quarkusio/quarkus/pull/31454[Hibernate Reactive 2]
+  * @Inject for https://github.com/quarkusio/quarkus/pull/31392[Hibernate StatelessSession]
+  * https://github.com/quarkusio/quarkus/pull/31175[Multiple Persistence Unit] support for Spring Data repositories
+  * New Developer Console
```
[the remark this answers is not in the document]
Comment (maxandersen) [reply _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:24]: yeah - my bad. Yes, Dev UI is what we should use :)

File: _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc [hunk 12]
```diff
@@ -0,0 +1,143 @@
...
+
+TBD: MTA command
+
+We want to improve and simplify these migration steps in preparation for Quarkus 3 thus do please https://github.com/quarkusio/quarkus/issues/new/choose[report] migration issues you encounter so we can improve the migration documentation and scripts.
+
+=== Hibernate Reactive Temporarily Disabled
+
+An early alpha version of Hibernate Reactive 2 is under active development and unfortunately not ready for use.
+
+In name of getting early feedback on all other parts of Quarkus we have excluded Hibernate Reactive from the Alpha5 relese. We are working hard on re-enabling it for an upcoming future release.
```
Comment (gsmet) [inline _posts/2023-03-08-quarkus-3-0-0-alpha5-released.adoc:49]:
```suggestion
In name of getting early feedback on all other parts of Quarkus we have excluded Hibernate Reactive from the Alpha5 release. We are working hard on re-enabling it for an upcoming future release.
```

**Pull request merged**